Categories/Morphism/IsoEquiv.agda

```diff
@@ -50,6 +52,9 @@ record _≃_ (i j : A ≅ B) : Set (o ⊔ ℓ ⊔ e) where
   ; isEquivalence = ≃-isEquivalence
   }
 
+-- sstucki: shouldn't this be called the `Core'?
```

Comment:
Yes indeed. Apparently neither of us had noticed that there was established terminology for this.  But that should be done separately.